src/public/parts/shells/cabinet/public/index.css

```
@@ -0,0 +1,81 @@
+	min-width: 100%;
+	gap: 0.25rem;
+}
+.cabinet-breadcrumb li {
+	display: flex;
+	align-items: center;
+	min-width: 0;
+}
+.cabinet-breadcrumb li:not(:last-child, .breadcrumb-back)::after {
+	content: "›";
+	margin-left: 0.25rem;
+	color: color-mix(in oklch, currentColor 35%, transparent);
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)